src/module.indexer/model/poolpair.ts

```diff
@@ -34,9 +34,10 @@ export class PoolPairIndexer extends Indexer {
             ? data.pairSymbol
             : await this.constructPairSymbol(data.tokenA.toString(), data.tokenB.toString())
 
-          const poolId = await this.newPoolId()
+          const rpcPoolPair = await this.client.poolpair.getPoolPair(pairSymbol)
```

Comment:
```suggestion
          const rpcPoolPair = await this.client.poolpair.getPoolPair(pairSymbol)
          if (Object.keys(rpcPoolPair).length !== 1) throw
```
Can put a guard here, the indexer will make noise if same pair allowed in defid from different operator in future.